src/vrp_worker.h

```diff
@@ -32,7 +32,7 @@ struct VRPWorker final : Nan::AsyncWorker {
             std::int32_t numVehicles_,                        //
             std::int32_t vehicleDepot_,                       //
             std::int32_t timeHorizon_,                        //
-            std::int32_t vehicleCapacity_,                    //
+            std::vector<int64> vehicleCapacity_,              //   type changed to vector int64
```

Comment:
it was because the AddDimensionWithVehicleCapacity()  takes long long integer array